aii-core/src/test/perl/grub2_efi_options.t

```diff
@@ -0,0 +1,72 @@
+use strict;
+use warnings;
+use CAF::Application;
+use Test::Quattor;
+use Test::More;
+use NCM::Component::PXELINUX::constants qw(:pxe_constants);
+use AII::Shellfe;
+use Readonly;
+
+Readonly my $AII_CONFIG_EMPTY => 'src/test/resources/aii-empty.conf';
+Readonly my $AII_CONFIG_NBPDIR => 'src/test/resources/aii-nbpdir.conf';
+Readonly my $AII_CONFIG_GRUB2_ONLY => 'src/test/resources/aii-grub2_only.conf';
+Readonly my $AII_CONFIG_PXELINUX_GRUB2 => 'src/test/resources/aii-pxelinux_grub2.conf';
+Readonly my $AII_CONFIG_PXELINUX_ONLY => 'src/test/resources/aii-pxelinux_only.conf';
+Readonly my $AII_LOG_FILE => 'target/test/aii-shellfe.log';
```

Comment:
Sorry, I missed that this was the AII log file, I thought it was some files produced by Maven...